runtime/common/src/lib.rs

```diff
@@ -170,37 +169,40 @@ impl WeightToFeeT for WeightToFee {
 
 /// Maps the reference time component of `Weight` to a fee.
 pub struct RefTimeToFee;
-impl WeightToFeePolynomial for RefTimeToFee {
+impl frame_support::weights::WeightToFeePolynomial for RefTimeToFee {
 	type Balance = Balance;
 
-	fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
-		// Map base extrinsic weight to 1/200 UNIT.
+	fn polynomial() -> frame_support::weights::WeightToFeeCoefficients<Self::Balance> {
+		// Map base extrinsic weight to 1/800 UNIT.
 		let p = UNIT;
-		let q = 200 * Balance::from(ExtrinsicBaseWeight::get().ref_time());
+		let q = 800
+			* Balance::from(
+				frame_support::weights::constants::ExtrinsicBaseWeight::get().ref_time(),
+			);
```

Comment:
I remembered that when we update this value, the corresponding gas price will also change, but the gas_price tests don't seem to fire, weird.

Reply:
```diff
+ assert_eq!(pallet_config::WeightPerGas::get().ref_time(), 75000);
```

Reply:
The increase in g here reduces the weight fee by 4 times. The increase in the block weight limit makes the WeightPerGas 4 times bigger. Ultimately, the gas price remains equal to the original one.